network/src/lib.rs

```diff
@@ -235,6 +271,28 @@ impl PolkadotProtocol {
 		rx
 	}
 
+	fn fetch_erasure_chunk(
+		&mut self,
+		ctx: &mut dyn Context<Block>,
+		relay_parent: Hash,
```

Comment:
suggest changing the argument types here to type aliases instead, as mentioned here https://github.com/paritytech/polkadot/pull/345/files#r308998575